lib/tasks/worldwide_taxonomy.rake

```diff
@@ -0,0 +1,26 @@
+namespace :worldwide_taxonomy do
+  task republish_world_locations: :environment do
+    DataHygiene::PublishingApiRepublisher.new(WorldLocation.all).perform
+  end
+
+  task redirect_world_location_translations_to_en: :environment do
+    base_path_prefix = "/government/world"
+    world_locations = WorldLocation.all
+
+    world_locations.each do |world_location|
+      en_slug = world_location.slug
+      destination_base_path = File.join("", base_path_prefix, en_slug)
+      content_id = world_location.content_id
+      locales = world_location.original_available_locales - [:en]
```

Comment:
Is `original_available_locales` a thing? It fails on my VM in the console:

```
irb(main):008:0> WorldLocation.first.original_available_locales
  WorldLocation Load (2.0ms)  SELECT  `world_locations`.* FROM `world_locations`  ORDER BY `world_locations`.`id` ASC LIMIT 1
NoMethodError: undefined method `original_available_locales' for #<WorldLocation:0x007f43bb565c60>
```

`available_locales` works:

```
irb(main):009:0> WorldLocation.first.available_locales
  WorldLocation Load (0.5ms)  SELECT  `world_locations`.* FROM `world_locations`  ORDER BY `world_locations`.`id` ASC LIMIT 1
  WorldLocation::Translation Load (0.1ms)  SELECT `world_location_translations`.* FROM `world_location_translations` WHERE `world_location_translations`.`world_location_id` = 1
=> [:en, :dr]
```